src/apis/study-room/study-room-info.ts

```diff
@@ -0,0 +1,16 @@
+import { http } from '../api';
+
+type Params = { studyId: number };
+type Response = {
+  id: number;
+  name: string;
+  intro: string;
+  step: string;
+  participantCount: number;
+  updateAt: string;
+};
```

Comment:
step 관련해서 상수 파일로 만들어주신게 있어서 이렇게 쓸 수 있는 상황이면 적용해주셔도 좋을 것 같네요~
```suggestion
type Response = {
  id: number;
  name: string;
  intro: string;
  step:Step;
  participantCount: number;
  updateAt: string;
};
```